peda.py

```diff
@@ -4792,14 +4795,17 @@ def eflags(self, *arg):
             eflags = peda.getreg("eflags")
             msg("%s: 0x%x (%s)" % (green("EFLAGS"), eflags, text.strip()))
 
-        if option == "set":
-            peda.set_eflags(flagname.lower())
```

Comment:
I think this was a bug: You had to type the flag lower case to clear it, but not to set it.